### ISSUE TITLE
‘service zigbee2mqtt upgrade’ don't upgrade to last release

### ISSUE THREAD

**Issue by tebra-jl** [paxswill/iocage-zigbee2mqtt#6]
I thing there is something missing in ‘service zigbee2mqtt upgrade’ because it don’t do ‘git checkout’ and ‘git pull’ commands. So the result is installation of the already existing version and not a upgrade.

**Comment by MArabie** [comment]
Agreed, I manually went to the `ZIGBEE2MQTT_DIR=/opt/zigbee2mqtt` and ran `git pull` then ran the `service zigbee2mqtt upgrade`. This built the updated pull.
The post_install.sh does a git clone, so you get the latest version as of installation.
I think @tebra-jl is correct and there should just be `git pull` added to line 33/34 of `/usr/local/etc/rc.d/zigbee2mqtt`.

I am not confident enough in my skills to do a pull request.  I am on the latest zigbee2mqtt now. I can attempt modifying the script on the next update. If this issue is still open and it works, I will pull request then.

**Comment by MArabie** [comment]
#9
